docs/getting-started/go.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Go
+---
+
+# Quickstart: ngrok-go
+
+---
+
+This quickstart will use the [ngrok-go Agent SDK](/agent-sdks/) to put a Go app
+on the internet and secure it so that only you can access it.
+
+## What is ngrok-go?
+
+ngrok-go is an [open-source, idiomatic Go
+package](https://github.com/ngrok/ngrok-go) for embedding secure ingress
+directly into your Go applications. If you’ve used the ngrok agent before, you
+can think of ngrok-go as the agent packaged as a Go library.
+
+ngrok-go enables you to serve Go applications on the internet in a single line
+of code without configuring network minutia like routing, load balancers,
+certificates, or ports. Applications using ngrok-go listen on ngrok’s global
+network and your app gets the same `net.Listener` interface it would get if it
+had listened on a local port. This makes it incredibly simple to drop ngrok-go
+into any existing Go code.
+
+You may want to open the [ngrok-go package documentation on
+pkg.go.dev](https://pkg.go.dev/golang.ngrok.com/ngrok) as you work through this
+guide.
+
+## Step 1: Install ngrok-go
+
+Create a new Go module and install ngrok-go. This guide assumes you [already
+have Go installed](https://go.dev/doc/install).
+
+```bash
+mkdir ngrok-go-quickstart
+cd ngrok-go-quickstart
+go mod init ngrok-go-quickstart
+go get golang.ngrok.com/ngrok
+```
+
+## Step 2: Create your app
+
+In the `ngrok-go-quickstart` directory, create the following `main.go` file:
+
+```go showLineNumbers
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	"golang.ngrok.com/ngrok"
+	"golang.ngrok.com/ngrok/config"
+)
+
+func main() {
+	if err := run(context.Background()); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func run(ctx context.Context) error {
+	// highlight-start
+	listener, err := ngrok.Listen(ctx,
+		config.HTTPEndpoint(),
+		ngrok.WithAuthtokenFromEnv(),
+	)
+	// highlight-end
+	if err != nil {
+		return err
+	}
+
+	log.Println("App URL", listener.URL())
+	// highlight-next-line
+	return http.Serve(tun, http.HandlerFunc(handler))
+}
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintln(w, "<h1>Hello from ngrok-go!</h1>")
+}
+```
+
+:::note In this code
+
+- **Lines 20-23**: connect to ngrok and create a net.Listener for a randomly-assigned HTTP endpoint URL
+- **Line 22**: use the value of the `NGROK_AUTHTOKEN` environment variable to auth with ngrok
```

Comment:
COMMENT: We should instruct them to set this environment variable above for a smoother experience.

Reply:
Should have read a little further. :)